tests/netlist/test_netlist_utils.py

```diff
@@ -0,0 +1,225 @@
+import pytest
+
+from nengo_spinnaker.utils.keyspaces import KeyspaceContainer
+from nengo_spinnaker.netlist import Net, Vertex, utils
+
+
+def test_get_nets_for_placement():
+    """Test that Rig nets can be generated to be used during the placement."""
+    # Create the vertices
+    a = [object(), object()]
+    b = [object(), object()]
+    c = object()
```

Comment:
This naming scheme is slightly confusing. If you can be bothered doing

```
a = object()
b = object()
c = object()
d = object()
e = object()

ab_cd = Net([a, b], [c, d], 1.0, None)
c_e = Net(c, e, 2.0, None)
```
etc. then it would (in my eyes!) make this test much more obvious!